internal/binutils/binutils.go

```diff
@@ -159,12 +228,15 @@ func findExe(cmd string, paths []string) (string, bool) {
 // of a binary.
 func (bu *Binutils) Disasm(file string, start, end uint64, intelSyntax bool) ([]plugin.Inst, error) {
 	b := bu.get()
-	args := []string{"-d", "-C", "--no-show-raw-insn", "-l",
-		fmt.Sprintf("--start-address=%#x", start),
+	if !(b.objdumpFound || b.llvmObjdumpFound) {
```

Comment:
I think `if !b.objdumpFound && !b.llvmObjdumpFound` would be easier to read, but maybe it's just me.

Reply:
This would be even simpler if instead of objdumpFound and llvmObjdumpFound we'd have objdumpFound and isLLVMObjdump where the first flag is for whether the tool is found and the second for what flavor it is.

Reply:
Done. Done.

Reply:
Done. Done.